src/tooltip/tooltip.js

```diff
@@ -174,7 +175,9 @@ angular.module('ui.bootstrap.tooltip', ['ui.bootstrap.position', 'ui.bootstrap.s
             // TODO add ability to start tooltip opened
             ttScope.isOpen = false;
             openedTooltips.add(ttScope, {
-              close: hide
+              close: hide,
+              triggerElement: element,
+              element: null
```

Comment:
My philosophy is don't add it unless it's needed.  It just adds unnecessary overhead to the openedTooltips collection.